src/components/VariableFontStyleEntry.tsx

```diff
@@ -5,18 +5,15 @@ import {
     Color,
     Flyout,
     FormControl,
-    IconCross,
     IconMinusCircle,
     IconPen,
     TextInput,
     Textarea,
 } from '@frontify/fondue';
-import React, { ChangeEvent, Dispatch, FC } from 'react';
-import usePromise from 'react-use-promise';
```

Comment:
Is it possible to remove this from the dependencies? If yes please do it!

Reply:
React, FC and Dispatch are still used in the code. I removed usePromise using npm uninstall.